### ISSUE TITLE
Skipper doesn't parse alle elements in an array (143 out of 250)

### ISSUE THREAD

**Issue by fabdrol** [balderdashy/skipper#77]
I'm POSTing batches (arrays) of 250 objects with location data to my API. For some reason, my controller only receives 143 elements, the rest seems to be lost in translation. When I check in web inspector, I can see that all 250 items really are sent by the client.

**Comment by metzuda** [comment]
Can you give us any more information? 

I'm posting a form with ~400 data points all in nested array form (e.g. ```name="formData[heading][subheading][title][data]"```) and it all comes out in good shape. 

if you are using enctype 'multipart/form-data' skipper doesn't seem to parse arrays, but only prepares simple JSON objects (e.g. ```{"formData[heading][subheading][title][data]" : "hello"}```; you'll have to do your own parsing in the controller.

**Comment by sailsbot** [comment]
Thanks for posting, @fabdrol.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
